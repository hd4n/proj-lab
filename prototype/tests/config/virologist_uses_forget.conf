@effects
fo f1 1

@agents
vi vi1 f1

@codes
co cd1 vi1 0 0
co cd2 vi1 0 0

@fields
la l1 cd1
em e2

@neighbors
l1 e2

@virologists
v1 e2 null null null
v2 l1 null null null