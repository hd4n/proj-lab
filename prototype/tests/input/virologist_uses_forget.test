load virologist_uses_forget.conf
getstats v2
craft v1 cd1 #kell?
useagent v1 v2 f1
getstats v2
