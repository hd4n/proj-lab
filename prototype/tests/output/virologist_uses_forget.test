Load: sikeres

stats: v2
	fi_e2
	di_null
	st_false
	res_0.0
	re_false
	ma_10
	ref_0
	neededAmino_-1
	neededNucleo_-1
	co_null
	ma_null
	ag_null
	eq_null
	eqa_null
	ef_null
craft: v1 cd1
useagent: v1 v2 vi1

stats: v2
	fi_e2
	di_null
	st_false
	res_0.0
	re_false
	ma_10
	ref_0
	neededAmino_-1
	neededNucleo_-1
	co_null
	ma_null
	ag_null
	eq_null
	eqa_null
	ef_f1
