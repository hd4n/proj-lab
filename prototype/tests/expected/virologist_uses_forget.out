Load: sikeres

stats v2:
fi_e2
eq_null
ma_null
ef_null
co_null

craft: c1 cd1 vi1 #kell?

useagent:
v1+v2
forget

stats v2:
fi_e2
eq_null
ma_null
ef_f1
co_null
